CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+### 1.7.0 3/4/2016
+* Improved the display of file metadata for mobile users
+* Prevent the use of script tags in all form fields
+* The page footer now diplays the date and time of the last code deploy
+* Added a link to the File Format Advice page at the top of the new work form
+* Users can now reserve Document Object Indentifiers (DOIs) for embargoed/private works
```

Comment:
A bonus of this is that they can now also update the metadata associated with their DOIs.